src/send/SendEnterAmount.test.tsx

```diff
@@ -357,17 +357,17 @@ describe('SendEnterAmount', () => {
   })
 
   it('shows warning when prepareTransactionResult type is not-enough-balance-for-gas', () => {
+    mockUsePrepareSendTransactionsOutput.prepareTransactionsResult = {
```

Comment:
these tests were failing after this refactor without this change. i'm not sure why, but it seems the right thing to do to set the mocks first before rendering the component. after the `render`, if we were to simulate the hook's output value changing, we'd also need to get the component to "listen" and react to it - i don't see the value of testing that part. having said that, i'm not sure why these tests were passing like this before...